### PR TITLE
Igor/dshb 462

### PR DESCRIPTION
**Pull request by igormas**

### CODE REVIEW

File: dist/contentAvail/contentAvail.js [hunk 1]
```diff
@@ -12,30 +12,31 @@
...
 
                 var PURCHASED_ALL = 'all';
 
                 var ContentAvailSrvObj = {};
 
-                function getUserPurchaseData(){
+                function getUserPurchaseData() {
                     var StorageService = InfraConfigSrv.getStorageService();
-                    var purchaseDataPath = StorageService.variables.appUserSpacePath + '/purchase';
+                    var purchaseDataPath = StorageSrv.variables.appUserSpacePath + '/purchase';
```
Comment (assafshp) [inline dist/contentAvail/contentAvail.js]: it was a bug?

Comment (igormas) [inline dist/contentAvail/contentAvail.js]: Nope

File: src/components/storage/storageSrv.js [hunk 2]
```diff
@@ -68,46 +72,99 @@
...
+                        angular.forEach(pathStrOrObj, function (value, pathName) {
+                            var processedPath = _replaceVariables(pathName, uid);
                             processedPathObj[processedPath] = value;
                         });
 
                         return processedPathObj;
                     }
                     $log.error('StorageSrv: failed to process path');
+
+                    return null;
```
Comment (assafshp) [inline src/components/storage/storageSrv.js]: till now did this function retuned something ?

Comment (igormas) [inline src/components/storage/storageSrv.js]: Yes, it returning the processed path (uid variable was replaced with real uid )

**Pull request closed without merging**